tools/test-builds/browser-agent-wrapper/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "browser-agent-wrapper",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js --mode=production --node-env=production"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@newrelic/browser-agent": "file:../../../temp/newrelic-browser-agent-0.1.229.tgz"
```

Comment:
Having to (remember) update the numbers here each version is going to be annoying, no?

Reply:
It automatically does it...